cmd/buildkitd/main.go

```diff
@@ -443,6 +443,12 @@ func setDefaultNetworkConfig(nc config.NetworkConfig) config.NetworkConfig {
 	if nc.CNIBinaryPath == "" {
 		nc.CNIBinaryPath = appdefaults.DefaultCNIBinDir
 	}
+	if nc.BridgeName == "" {
+		nc.BridgeName = "buildkit0"
+	}
+	if nc.BridgeSubnet == "" {
+		nc.BridgeSubnet = "10.10.0.0/16"
```

Comment:
In a follow up PR, could you add a documentation to explain this default CIDR and other configuration?